src/test/java/net/openhft/chronicle/bytes/BytesJavaDocComplianceTest.java

```diff
@@ -33,15 +34,17 @@
     @BeforeEach
     void beforeEach() {
         if (INITIAL_INFO_MAP.isEmpty()) {
+            AtomicInteger count = new AtomicInteger(0);
             provideBytesObjects()
                     .forEach(args -> {
+                        count.incrementAndGet();
                         final Bytes<Object> bytes = bytes(args);
                         INITIAL_INFO_MAP.put(createCommand(args), new BytesInitialInfo(bytes));
                         releaseAndAssertReleased(bytes);
                         Jvm.pause(100);
                     });
             // Make sure we have unique keys
-            assertEquals(provideBytesObjects().count(), INITIAL_INFO_MAP.size());
```

Comment:
There is was @nicktindall . Well done and thanks ever so much.

Reply:
You're very welcome @minborg